<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmHydraulicsOptions</class>
 <widget class="QMainWindow" name="frmHydraulicsOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>358</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>EPANET Hydraulics Options</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblFlowUnits">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Flow Units</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblHeadloss">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Headloss Formula&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSpecificGravity">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Specific Gravity</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdOK">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdCancel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboFlow">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cboHeadloss">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSpecificGravity">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblRelativeViscosity">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Relative Viscosity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtRelativeViscosity">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblMaximumTrials">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum Trials</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMaximumTrials">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblAccuracy">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Accuracy</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAccuracy">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblDefaultPattern">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Default Pattern</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDemandMultiplier">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Demand Multiplier</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEmitterExponent">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Emitter Exponent</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDefaultPattern">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDemandMultiplier">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEmitterExponent">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblCheckFrequency">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Check Frequency</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMaxCheck">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Check</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDampLimit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Damp Limit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCheckFrequency">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMaxCheck">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDampLimit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbxUnbalanced">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>If Unbalanced</string>
    </property>
    <widget class="QLineEdit" name="txtContinueN">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbnStop">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbnContinue">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbnContinueN">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Continue N</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbxHydraulics">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Hydraulics File</string>
    </property>
    <widget class="QLineEdit" name="txtHydraulicsFile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbnUse">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Use</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbnSave">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
